MAUI/Calendar/customizations.md

```diff
@@ -222,4 +291,352 @@ You can customize the selection cell background and text style in Month, Year, D
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+    <Grid.Resources>
+        <DataTemplate x:Key="normalDateTemplate">
+            <Grid Background = "Pink" >
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Day}"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="todayDateTemplate">
+            <Grid Background = "PaleGreen">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Day}"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="leadingTraililngDateTemplate">
+            <Grid Background = "Purple">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Day}"/>
+            </Grid>
+        </DataTemplate>
+        <local:MonthCellTemplateSelector x:Key="monthCellTemplateSelector" 
+                                         TodayDateTemplate="{StaticResource todayDateTemplate}" 
+                                         NormalDateTemplate="{StaticResource normalDateTemplate}" 
+                                         LeadingTrailingDateTemplate="{StaticResource leadingTraililngDateTemplate}"/>
+    </Grid.Resources>
+    <calendar:SfCalendar x:Name="Calendar" 
+                        View="Month" >
+        <calendar:SfCalendar.MonthView>
+            <calendar:CalendarMonthView CellTemplate="{StaticResource monthCellTemplateSelector}" />
+        </calendar:SfCalendar.MonthView>
+    </calendar:SfCalendar>
+    </Grid>
+
+{% endhighlight %}
+{% highlight c# tabtitle="TemplateSelector.cs" %}
+
+public class MonthCellTemplateSelector : DataTemplateSelector
+{
+    public MonthCellTemplateSelector()
+    {
+    }
+    public DataTemplate NormalDateTemplate { get; set; }
+    public DataTemplate TodayDateTemplate { get; set; }
+    public DataTemplate LeadingTrailingDateTemplate { get; set; }
+
+    protected override DataTemplate OnSelectTemplate(object item, BindableObject container)
+    {
+        var monthCellDetails = item as CalendarCellDetails;
+        if (monthCellDetails.Date == DateTime.Today.Date)
+            return TodayDateTemplate;
+        else if (monthCellDetails.IsTrailingOrLeadingDate)
+            return LeadingTrailingDateTemplate;
+        else
+            return NormalDateTemplate;
+    }
+}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Year cell appearance using DataTemplate
+
+The year cell appearance can be customized using the `CellTemplate` property of `YearView` in the `SfCalendar`.
```

Comment:
The year cell appearance can be customized using the `CellTemplate` property of `YearView` in the `SfCalendar.`